docs/network-flow-visibility.md

```diff
@@ -259,7 +259,7 @@ modes of operation:
   statelessly, and the flow records received from the Flow Exporter of Antrea
   Agents are sent directly to an IPFIX collector, without buffering or
   correlation / aggregation. For more information about this mode, including
-  installation instructions, refer to the [Proxy Mode section](#proxy-mode).
+  installation instructions, refer to the [Proxy Mode section](#proxy-mode-v23-and-above).
```

Comment:
@wenyingd please rebase the code since a fix is ready merged. https://github.com/antrea-io/antrea/pull/7032